doc/sphinx/source/notes_7_0.rst

```diff
@@ -21,9 +38,58 @@ CPU drivers
   Please note that these DBKs are still a proof-of-concept and
   are subject to change without notice.
 
+* When built with -DENABLE_CONFORMANCE=ON, PoCL's CPU driver
+  is now able to pass the full OpenCL-CTS conformance testsuite.
+  We have submitted & achieved conformant status on <TODO day>.
+
+* Implemented Windows platform support, using both MinGW and MSVC
```

Comment:
Reimplemented. We used to support it some time ago to some extent. Also a release highlight.

Reply:
Should go to "General Features". Both the CPU and LZ drivers should work in Windows.